### PR TITLE
sahil.json

### PR DESCRIPTION
**Pull request by Sahilkhanbossking**
{

  "owner": {
    "username": "Sahil@1992",
    "email": "s4h1l.ml"
  },

  "target": {
    "CNAME": {
       "name": "S4h1l.cf, 
       "value": ""
     }
    "txt": {
    "value": "replit-verify=40ad71ed-509d-4dd9-9c0c-dfb5e0725222"
  },

  "proxied": false
}

### CODE REVIEW

**Comment by KaminoCodes** [comment]
Multiple things:
> A valid email address is required
> No cname value is supplied

File: sub-logs/sahil.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+
+  "owner": {
+    "username": "Sahilkhanbossking",
+    "email": ""
```
Comment (WilliamDavidHarrison) [inline sub-logs/sahil.json:5]: Please specify an email

File: sub-logs/sahil.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+
+  "target": {
+    "CNAME": {
+       "name": "sahil", 
+       "value": "s4h1l.cf"
+     },
+
+     "TXT": {
+       "name": "",
```
Comment (WilliamDavidHarrison) [inline sub-logs/sahil.json:15]: Please add the  name of the TXT record.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Comment by DIBSTERYT** [comment]
@Sahilkhanbossking Reminder if you want this merged, we ask you fill out/fix said issues with your pull request. 👍

**Comment by wclarkey** [comment]
If this is not solved in 24-48 hours, this pull request will be closed.

**Comment by DIBSTERYT** [comment]
As per comment above, this pull request is now closed.

**Pull request closed without merging**